cdap-operational-stats-core/src/test/java/co/cask/cdap/operations/cdap/CDAPOperationalStatsTest.java

```diff
@@ -91,9 +91,10 @@ public void test() throws Exception {
     Assert.assertEquals(1, transactions.getNumInProgressTransactions());
     Assert.assertEquals(1, transactions.getNumInvalidTransactions());
     Assert.assertEquals(1, transactions.getNumCommittingChangeSets());
-    // the entites.collect calls appLifecycleService.getApps this fetches the owner principal of the deployed which
-    // makes the committed transaction 1 here
-    Assert.assertEquals(1, transactions.getNumCommittedChangeSets());
+    // the entites.collect calls appLifecycleService.getApps which fetches the owner principal of the deployed
+    // app making the committed transaction 1 here. We assert for 1 or greater because we might add other such calls
+    // which will cause a transaction to be completed.
+    Assert.assertTrue(transactions.getNumCommittedChangeSets() >= 1);
```

Comment:
``>= 0``, right - since this particular branch added a committed changeset? Also, can we make the 3 before these (lines 91-93) also ``>= 1``?